### ISSUE TITLE
unused variable, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1198]
```abap
DATA rb_langa TYPE c.
DATA(language) = COND #( WHEN rb_langa = abap_true THEN '%' ELSE 'a' ).
WRITE language.
```
